ebpf/event_test.go

```diff
@@ -79,72 +79,63 @@ func BenchmarkUniqueConnKeyByteBufferPacked(b *testing.B) {
 		// PID (32 bits) + SPort (16 bits) + DPort (16 bits) = 64 bits
 		p0 := uint64(c.Pid)<<32 | uint64(c.SPort)<<16 | uint64(c.DPort)
 		binary.Write(buf, binary.LittleEndian, p0)
-		buf.WriteString(c.Source)
+		buf.WriteString(c.SourceAddr().String())
 		// Family (8 bits) + Type (8 bits) = 16 bits
 		p1 := uint16(c.Family)<<8 | uint16(c.Type)
 		binary.Write(buf, binary.LittleEndian, p1)
-		buf.WriteString(c.Dest)
+		buf.WriteString(c.DestAddr().String())
 		buf.Bytes()
```

Comment:
Does this work?  I thought the compiler would optimize this away

Reply:
I'm going to delete these benchmarks. They're really old and don't reflect the current state of the byte key generation